src/config.rs

```diff
@@ -894,6 +894,17 @@ mod tests {
         assert_eq!(got.html_config().unwrap(), html_should_be);
     }
 
+    #[test]
+    #[should_panic(expected = "Invalid configuration file")]
+    fn fail_on_invalid_config_field() {
+        let src = r#"
+        [book]
+        password = Secret
+        "#;
+
+        Config::from_str(src).unwrap();
+    }
```

Comment:
I don't think this test is actually checking the change in this PR. It seems to be checking that the TOML syntax is invalid.